lib/rubocop/cop/chef/correctness/incorrect_library_injection.rb

```diff
@@ -36,10 +38,18 @@ class IncorrectLibraryInjection < Cop
             (send (const (const (cbase) :Chef) {:Recipe :Provider}) :send (sym :include) ... )
           PATTERN
 
+          def_node_matcher :legacy_class_includes?, <<-PATTERN
+            (send (const (const (cbase) :Chef) {:Recipe :Provider}) :include ... )
+          PATTERN
```

Comment:
Probably not as big of a problem, but this could also look for `extend` and `prepend`.

Reply:
Quick search of supermarket isn't showing any uses of those so I think we're safe